library.properties

```diff
@@ -1,5 +1,5 @@
 name=STM32duino X-NUCLEO-NFC04A1
-version=1.0.2
+version=1.1.0
 author=STMicroelectronics
 maintainer=stm32duino
 sentence=Allows controlling the ST X-NUCLEO-NFC04A1 expansion board
```

Comment:
Hi Carlo,
don't forget to add the dependencies in the library.properties before the next release when the ST25DV will be referenced by Arduino.
Thanks

Reply:
Hi @fpistm ,
I have already updated the NFC04 library with the update of the library version and the dependency. I'm waiting for the official publication of the ST25DV library before tagging the repo.
Best Regards,
Carlo